RepositoryCompiler/Communication/MetricsDTO.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using RepositoryCompiler.CodeModel.CaDETModel.Metrics;
+
+namespace RepositoryCompiler.Communication
+{
+    public class MetricsDTO
+    {
+        public int NOP { get; set; }
+
+        public int CYCLO { get; set; }
+
+        public int NOLV { get; set; }
+
+        public int LOC { get; set; }
+
+        public MetricsDTO() { }
+
+        public MetricsDTO(CaDETClassMetrics parsedClassMetrics)
+        {
+            LOC = parsedClassMetrics.LOC;
```

Comment:
Yes, this is incomplete because the focus was to activate AMQP communication from **plugin** and send real class which is just parsed (not some mockup like at the tests).

We will work on this furthermore.